evgMrmApp/src/evgInit.cpp

```diff
@@ -108,7 +108,7 @@ static const evgMrm::Config conf_mtca_evm_300 = {
     "mTCA-EVM-300",
     3,
     16,
-    0,
+    16,
```

Comment:
MTCA RTM supports only 10 inputs but 16 is used to avoid initialization errors. To address that, I suggest handling it in a separate PR.

Reply:
Sounds good to me. Thanks!